src/js/markdown-editor.js

```diff
@@ -1,12 +1,53 @@
 import 'easymde/dist/easymde.min.css';
 import EasyMDE from 'easymde';
+import apiFetch from './apiFetch'
+
+const _form = document.querySelector('[data-s3-photo="form"]');
+
+const createPhoto = (imageUrl) => {
+  const objectUuid = _form.dataset.s3UppyObjectUuid;
+  const photoType = _form.dataset.s3UppyPhotoType;
+  // Create model for this user with s3 image url
+  return apiFetch('/api/photos', {
+    data: { photo: { direct_url: imageUrl, photo_type: photoType, object_uuid: objectUuid } }
+  });
+};
+
+const getXmlValue = (source, tagName) => {
+  const start = source.indexOf(`<${tagName}>`)
+  const end = source.indexOf(`</${tagName}>`, start)
+  return start !== -1 && end !== -1
+    ? source.slice(start + tagName.length + 2, end)
+    : ''
+};
 
 const markdownEditor = document.querySelector('[data-markdown-editor]');
 const easyMDE = new EasyMDE({
   element: markdownEditor,
-  showIcons: ["code"],
-  status: false
+  // TODO: fix preview, and uncomment preview icon
```

Comment:
Is there a Trello card for this? :>

Reply:
Nope, we will create it when there will be actual need for this feature.